conf/application.conf

```diff
@@ -69,6 +58,11 @@ microservice {
       host = localhost
       port = 8311
     }
+    emcs-tfe-reference-data {
+      protocol = http
+      host = localhost
+      port = 8312
+    }
```

Comment:
There is an entry already for `reference-data`. Could you remove that entry please?

Reply:
(and replace any occurrences of `reference-data` with `emcs-tfe-reference-data`)

Reply:
Fixed in https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/903817e690ffef14174a5b82fddc305d93662fd6